src/XRP/BinaryCoding.h

```diff
@@ -47,4 +56,10 @@ inline void encodeBytes(std::vector<uint8_t> bytes, std::vector<uint8_t>& data)
     data.insert(data.end(), bytes.begin(), bytes.end());
 }
 
+inline void encode0(int len, std::vector<uint8_t>& data) {
+    for (int i = 0; i < len; i++) {
+        data.push_back(0);
+    }
+}
```

Comment:
Here I suggest extending the vector using `reserve` and using `https://en.cppreference.com/w/cpp/algorithm/fill_n` - len doesn't need to be negative, switching to a `std::size_t` is recommended.

Reply:
I'll use TW::Append() per catenocrypt's suggestion